starcounter-include.html

```diff
@@ -476,13 +520,28 @@
          * @param {DocumentFragment} givenComposition
          */
         StarcounterIncludePrototype.stampComposition = function(givenComposition){
-            // create shadowRoot if none
             const shadowRoot = this.shadowRoot;
+            this.blockingShadowLinksLoaded && this.blockingShadowLinksLoaded.reject();
```

Comment:
If I comment out this line, all tests still pass. What is the purpose of it?

Reply:
I'm trying to write a test for it.